service/dap/server.go

```diff
@@ -778,9 +822,12 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 			}
 		}
 
-		s.log.Debugf("building binary at %s", debugbinary)
 		var cmd string
 		var out []byte
+		// Log debug binary build
+		if !((mode == "replay") && (s.config.Debugger.Backend == "core")) {
```

Comment:
This if is only entered for "debug" or "test". There is nothing to build in replay/core mode, is there?

Reply:
Indeed, that´s why the log is omitted in these cases

Reply:
Yes, my bad, I corrected it to `!((mode == "replay") || (s.config.Debugger.Backend == "core"))`